python/mxnet/gluon/estimator/estimator.py

```diff
@@ -208,16 +203,15 @@ def evaluate(self,
     def fit(self, train_data,
             val_data=None,
             epochs=1,
-            batch_size=None,
             event_handlers=None,
             batch_fn=None):
         """Main training loop
```

Comment:
Can you add what Fit does here and also note that the batch_size is inferred from the dataloader's batch_size.

Reply:
Done. Thanks!